packages/accounts-app/src/Accounts/BalanceOverview.tsx

```diff
@@ -146,9 +147,9 @@ export function BalanceOverview (props: Pick<Props, Exclude<keyof Props, keyof '
                 </WithSpace>
               : renderUnBondedAccountOptions()
         }
-        <WithSpace><SubHeader>Stash Active:</SubHeader> <FadedText>{stakingLedger && formatBalance(stakingLedger.active)}</FadedText> </WithSpace>
-        <WithSpace><SubHeader>Stakers Total:</SubHeader> <FadedText>{formatBalance(stakers && stakers.total)}</FadedText> </WithSpace>
-        <WithSpace><SubHeader>Bonded:</SubHeader> <FadedText>{stakingLedger && formatBalance(stakingLedger.total)} </FadedText></WithSpace>
+        <WithSpace><SubHeader>Stash Active:</SubHeader> <FadedText>{stakingLedger && formatBalance(stakingLedger.active.toBn())}</FadedText> </WithSpace>
```

Comment:
You can skip the `toBn` on `formatBalance` (assuming `@polkadot/util` `^1.0.1` - for compacts it auto-unwraps when creating these)